cub/test/catch2_test_launch_wrapper.cu

```diff
@@ -67,8 +67,10 @@ struct cub_api_example_t
                                                  const T *d_in,
                                                  T *d_out,
                                                  int num_items,
-                                                 bool should_be_invoked_on_device)
+                                                 cudaStream_t stream = 0)
   {
+    constexpr bool should_be_invoked_on_device = TEST_LAUNCH == 1;
+
     NV_IF_TARGET(NV_IS_HOST,
                  (if (should_be_invoked_on_device) { return cudaErrorLaunchFailure; }),
                  (if (!should_be_invoked_on_device) { return cudaErrorLaunchFailure; }));
```

Comment:
That line of code is super confusing as it is difficult to discern that this is actually an NV_IF_ELSE_TARGET

Could you change the `NV_IF_TARGET` to `NV_IF_ELSE_TARGET`